_layouts/post.html

```diff
@@ -3,6 +3,7 @@
 ---
 <article class="single-post {{ page.collection }}" itemscope itemtype="http://schema.org/BlogPosting" role="article">
   {% include page-header.html %}
+  {% include header-custom-viz.html %}
```

Comment:
Please look at the Ocean example more closely. The includes to include the extra CSS & JS are done in the `head.html` and `footer-scripts.html` includes.

Reply:
Thank you, @jnschrag. Looking at the work I did on this last week – a lot of it totally doesn't even make sense to me. I fixed a couple things, and am going to hack away at it. I'll be able to dedicate more of my day to it starting on Thursday (which maybe could be my 'viz day' that we had talked about previously?). Tuesday and Wednesday seem to be more broken up with meetings, etc.